connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/PluginUtils.java

```diff
@@ -453,4 +457,58 @@ private static class DirectoryEntry {
         }
     }
 
+    private static Collection<URL> forJavaClassPath() {
+        Collection<URL> urls = new ArrayList<URL>();
```

Comment:
nit: The second `<URL>` is not required and can be `<>`

Reply:
Done.